wm-system/src/main/kotlin/com/ajaxproject/warehouse/config/KafkaConfig.kt

```diff
@@ -20,6 +26,10 @@ class KafkaConfig(
     val keySerializer: String,
     @Value("\${spring.kafka.producer.value-serializer}")
     val valueSerializer: String,
+    @Value("\${spring.kafka.consumer.key-deserializer}")
+    val keyDeserializer: String,
+    @Value("\${spring.kafka.consumer.value-deserializer}")
+    val valueDeserializer: String,
     @Value("\${spring.kafka.registry.url}")
     val registryUrl: String
```

Comment:
In my case it is a little more complicated because of more nesting and different prefixes, so to implement this, I will need to create 3 additional config classes (for producer, for consumer and general props for Kafka)